File: Sandwitch.Portal/Sandwitch.Tier.ViewModels/Classes/Additions/AddHistorico.cs
using System;

namespace Sandwitch.Tier.ViewModels.Classes.Additions
{
    public class AddHistorico
    {
        public AddHistorico()
        {
        }

        public int ArenalId { get; set; }

        public int BanderaId { get; set; }

        public double Temperatura { get; set; }

        public DateTime BajaMarAlba { get; set; }

        public DateTime BajaMarOcaso { get; set; }

        public DateTime AltaMarAlba { get; set; }

        public DateTime AltaMarOcaso { get; set; }
    }
}

File: Sandwitch.Client/src/viewmodels/additions/addhistorico.ts
import { Time } from "@angular/common";

export interface AddHistorico {
    ArenalId: number;
    BanderaId: number;
    VientoId: number;
    Velocidad: number;
    Temperatura: number;
    BajaMarAlba: Time;
    BajaMarOcaso: Time;
    AltaMarAlba: Time;
    AltaMarOcaso: Time;
}
